conda/cli/main_info.py

```diff
@@ -118,30 +118,14 @@ def print_package_info(packages):
 
 def get_info_dict(system=False):
     try:
-        from requests import __version__ as requests_version
-
-        # These environment variables can influence requests' behavior, along with configuration
-        # in a .netrc file
-        #   CURL_CA_BUNDLE
-        #   REQUESTS_CA_BUNDLE
-        #   HTTP_PROXY
-        #   HTTPS_PROXY
-    except ImportError:  # pragma: no cover
-        try:
-            from pip._vendor.requests import __version__ as requests_version
-        except Exception as e:  # pragma: no cover
-            requests_version = "Error %r" % e
-    except Exception as e:  # pragma: no cover
-        requests_version = "Error %r" % e
-
-    try:
-        import conda_build
-    except ImportError:  # pragma: no cover
+        from conda_build import __version__ as conda_build_version
+    except ImportError as err:
+        # ImportError: conda-build is not installed
+        log.debug("Unable to import conda-build: %s", err)
         conda_build_version = "not installed"
-    except Exception as e:  # pragma: no cover
-        conda_build_version = "Error %s" % e
-    else:  # pragma: no cover
-        conda_build_version = conda_build.__version__
+    except Exception as err:
+        log.error("Error importing conda-build: %s", err)
```

Comment:
Could we remove the special ImportError handling and keep except Exception

Reply:
of course we can, I'm indifferent either way

Reply:
actually they indicate different kinds of information that gets displayed in `conda info` so I think its best to leave this

Reply:
I mean could they be combined to make code coverage testing easier? I see that printing the exception is one way it shows the difference between import and other errors.